### PR TITLE
FIx link to .net Corever and module template desc

### PR DESCRIPTION
**Pull request by mvktsk**
1. In lesson1.md changed link to direct download .NET Core Runtime v2.2.0 
2. In creating-new-module.md changed link to download the new version of Virto Commerce 2.x Module Templates + changed module solution structure description

### CODE REVIEW

File: pages/docs/lessons/lesson1.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Install-WindowsFeature -name Web-Server -IncludeAllSubFeature
 * [Microsoft SQL Server 2008 or later](https://www.microsoft.com/en-us/evalcenter/evaluate-sql-server-2017-rtm) with SQL Management Studio(free MS SQL Express would be enough)
 * [Visual C++ Redistributable Packages for Visual Studio](https://go.microsoft.com/fwlink/?LinkId=746572)
 * [Prerequisites for .NET Core on Windows](https://docs.microsoft.com/en-us/dotnet/core/windows-prerequisites)
-* [.NET Core Runtime](https://dotnet.microsoft.com/download)
+* [.NET Core Runtime v2.2.0](https://dotnet.microsoft.com/download/dotnet-core/thank-you/runtime-aspnetcore-2.2.0-windows-hosting-bundle-installer)
```
Comment (eMazeika) [inline pages/docs/lessons/lesson1.md]: 2.2.0 is 1 year old. Let's use the latest available:
`[.NET Core Runtime v2.2.8](https://dotnet.microsoft.com/download/dotnet-core/thank-you/runtime-aspnetcore-2.2.8-windows-hosting-bundle-installer)
`

File: pages/docs/lessons/lesson1.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ Install-WindowsFeature -name Web-Server -IncludeAllSubFeature
 * [Microsoft SQL Server 2008 or later](https://www.microsoft.com/en-us/evalcenter/evaluate-sql-server-2017-rtm) with SQL Management Studio(free MS SQL Express would be enough)
```
Comment (eMazeika) [inline pages/docs/lessons/lesson1.md]: `* Microsoft SQL Server 2017 or later or [SQL Server 2017 Express edition](https://www.microsoft.com/en-us/sql-server/sql-server-editions-express) with SQL Server Management Studio`

File: pages/docs/lessons/lesson1.md (hunk 1 above)
Comment (mvktsk) [reply pages/docs/lessons/lesson1.md]: fixed

File: pages/docs/lessons/lesson1.md (hunk 2 above)
Comment (mvktsk) [reply pages/docs/lessons/lesson1.md]: fixed

**Pull request merged**